docs/features/job-queue.txt

```diff
@@ -4,63 +4,135 @@
-*********************
+WebUI
+=====
 
-Users may modify the job queue in either the WebUI or the CLI.
+To modify the job queue in the Webui,
+
+#. Go to the **Job Queue** section.
+#. Find the job to modify.
+#. Click the three dots in the right-most column of the job.
+#. Find and click the **Manage Job** option.
+#. Make the change you want on the pop up page, and click **OK**.
+
+CLI
+===
+
+To modify the job queue in the CLI, use the ``det job update`` command. run ``det job update --help``
+for more information. Example operations:
+
+.. code::
+
+   $ det job update jobID --priority 10
+   $ det job update jobID --resource-pool a100
+   $ det job update jobID --ahead-of jobID-2
+
+To update a job in batch, provide updates as shown:
+
+.. code::
+
+   $ det job update-batch job1.priority=1 job2.resource-pool="compute" job3.ahead-of=job1
+
+Example workflow:
+
+.. code::
+
+   $ det job list
+      # | ID       | Type            | Job Name   | Priority | Submitted            | Slots (acquired/needed) | Status          | User
+   -----+--------------------------------------+-----------------+--------------------------+------------+---------------------------+---------
+      0 | 0d714127 | TYPE_EXPERIMENT | first_job  |       42 | 2022-01-01 00:01:00  | 1/1                     | STATE_SCHEDULED | user1
+      1 | 73853c5c | TYPE_EXPERIMENT | second_job |       42 | 2022-01-01 00:01:01  | 0/1                     | STATE_QUEUED    | user1
+
+   $ det job update 73853c5c --ahead-of 0d714127
+
+   $ det job list
+      # | ID       | Type            | Job Name   | Priority | Submitted            | Slots (acquired/needed) | Status          | User
+   -----+--------------------------------------+-----------------+--------------------------+------------+---------------------------+---------
+      0 | 73853c5c | TYPE_EXPERIMENT | second_job |       42 | 2022-01-01 00:01:01  | 1/1                     | STATE_SCHEDULED | user1
+      1 | 0d714127 | TYPE_EXPERIMENT | first_job  |       42 | 2022-01-01 00:01:00  | 0/1                     | STATE_QUEUED    | user1
+
+   $ det job update-batch 73853c5c.priority=1 0d714127.priority=1
+
+   $ det job list
+      # | ID       | Type            | Job Name   | Priority | Submitted            | Slots (acquired/needed) | Status          | User
+   -----+--------------------------------------+-----------------+--------------------------+------------+---------------------------+---------
+      0 | 73853c5c | TYPE_EXPERIMENT | second_job |       1 | 2022-01-01 00:01:01  | 1/1                     | STATE_SCHEDULED | user1
+      1 | 0d714127 | TYPE_EXPERIMENT | first_job  |       1 | 2022-01-01 00:01:00  | 0/1                     | STATE_QUEUED    | user1
 
 ***********************
- Job Queue Limitations
+ Job Queue Constraints
 ***********************
 
-.. code:: markdown
-
-   TODO
-   -  define and describe different states and their meanings
+Using the job queue to modify jobs has a number of constraints:
```

Comment:
I think "There are a number of constraints associated with using the job queue to modify jobs:" might sound a little better.